roles/edpm_iscsid/meta/argument_specs.yml

```diff
@@ -3,4 +3,34 @@ argument_specs:
   # ./roles/edpm_iscsid/tasks/main.yml entry point
   main:
     short_description: The main entry point for the edpm_iscsid role.
-    options: {}
+    options:
+      edpm_iscsid_debug:
+        type: bool
+        default: false
+        description: |
+          Produce additional text messages describing role operation.
+          Set to false by default using following template:
+          {{ (ansible_verbosity | int) >= 2 | bool }}
+      edpm_iscsid_image:
+        type: str
+        default: "quay.io/podified-antelope-centos9/openstack-iscsid:current-podified"
+        description: URL of the iscsid container image.
+      edpm_iscsid_config_dir:
+        type: path
+        default: /var/lib/config-data/ansible-generated/iscsid
```

Comment:
Description is missing, but this var isn't used at all. I'd remove it.

Directory is created [here](https://github.com/openstack-k8s-operators/edpm-ansible/blob/670640e21f4840408a7f3be26008733556e0fce6/roles/edpm_iscsid/tasks/install.yml#L19-L32), without referring to this directory.